.github/ISSUE_TEMPLATE/documentation.yml

```
@@ -0,0 +1,30 @@
+name: Documentation
+description: Report problems and suggest ideas for the documentation
+labels: [docs]
+
+body:
+  - type: markdown
+    attributes:
+      value: Before opening a new issue, please search for duplicate or closed issues.
+
+  - type: dropdown
+    attributes:
+      label: Kind of issue
+      description: Skip to "other" if you do not see your issue type here.
+      options: [improvement, mistake]
+
+  - type: input
+    attributes:
+      label: Kind of issue (other)
+      description: Only fill out if your issue type is not in the dropdown above.
```

Having the two separate inputs for this feels very awkward to me. It's a bit hard to reproduce the "feel" of the old template with the (understandably) limited schema GitHub offers though. I propose something like the following as a compromise, what do you think?

```suggestion
  - type: dropdown
    attributes:
      label: Kind of issue
      description: |
        Select "other" if you don't see your issue kind here. If you do, please explain
        why in the description below.
      options: [improvement, mistake, other]
```

---

Sounds good to me.